lib/chef/resource/windows_user_privilege.rb

```diff
@@ -137,10 +137,20 @@ class WindowsUserPrivilege < Chef::Resource
         required: true,
         coerce: proc { |v| v.is_a?(String) ? Array[v] : v },
         callbacks: {
-           "Option privilege must include any of the: #{privilege_opts}" => lambda { |v|
-             (privilege_opts & v).size == v.size
-           },
-         }
+          "Option privilege must include any of the: #{PRIVILEGE_OPTS}" => lambda { |n|
+            if n.is_a?(String)
+              these_options = Array[n]
+            else
+              these_options = n
+            end
+
+            if (these_options - PRIVILEGE_OPTS).empty?
+              true
+            else
+              false
+            end
```

Comment:
```suggestion
            (Array(n) - PRIVILEGE_OPTS).empty?
```

`Array()` is a function that will convert its parameter to an array. `Array[]` is the same as bare `[]`, that is, it creates an array containing its parameters.

Reply:
If the `coerce` proc is run first the conversion might not be necessary.

Reply:
You're too fast here @phiggins. I already have those same changes locally. This isn't done